hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -237,6 +237,7 @@ message ContainerInfoProto {
     optional ReplicationFactor replicationFactor  = 10;
     required ReplicationType replicationType  = 11;
     optional ECReplicationConfig ecReplicationConfig = 12;
+    optional bool isAllReplicaEmpty = 13 [default = true];
```

Comment:
Similar comment as above, I don't think we should persist this value as part of the main container state in SCM. And it *definitely* should not default to true.